packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/attachLine/telephony-line-phone-attach.html

```diff
@@ -0,0 +1,102 @@
+<div class="telecom-telephony-line-phone-attach">
+    <header>
+        <oui-back-button
+            data-ui-sref="telecom.telephony.billingAccount.line.phone"
```

Comment:
Could this be extracted to the routing's resolve to abstract use of ui router in the view